docs/man/monerod.1

```diff
@@ -0,0 +1,5343 @@
+.RE
+
+
+
+.SS /update
+
+Update daemon.
+
+.TP
+\fBAlias:\fR
+None.
+
+.TP
+\fBInputs:\fR
+
+\[bu] \fBcommand\fR - String; command to use, either \fBcheck\fR or \fBdownload\fR
+
+\[bu] \fBpath\fR - String; Optional, path where to download the update.
+
+
+.TP
+\fBOutputs:\fR
+
+\[bu] \fBauto_uri\fR - string;
+
+\[bu] \fBhash\fR - string;
+
+\[bu] \fBpath\fR - String; path to download the update
+
+\[bu] \fBstatus\fR - string; General RPC error code. "OK" means everything looks good.
+
+\[bu] \fBuntrusted\fR - boolean; States if the result is obtained using the bootstrap mode, and is therefore not trusted (\fBtrue\fR), or when the daemon is fully synced and thus handles the RPC locally (\fBfalse\fR)
+
+\[bu] \fBupdate\fR - boolean; States if an update is available to download (\fBtrue\fR) or not (\fBfalse\fR)
+
+\[bu] \fBuser_uri\fR - string;
+
+\[bu] \fBversion\fR - string; Version available for download.
+
+
+.TP
+.B Example:
+
+.nf
+.RS 4
+$ curl http://127.0.0.1:18081/update -d '{"command":"check"}' -H 'Content-Type: application/json'
+
+{
+  "auto_uri": "",
+  "hash": "",
+  "path": "",
+  "status": "OK",
+  "untrusted": false,
+  "update": false,
+  "user_uri": "",
+  "version": ""
+}
+
+.RE
+.fi
+
+
+.SS /pop_blocks
+
+.TP
+\fBAlias:\fR
+None.
+
+.TP
+\fBInputs:\fR
+
+\[bu] \fBnblocks\fR - unsigned int;
+
+
+.TP
+\fBOutputs:\fR
+
+\[bu] \fBheight\fR - unsigned int; New blockheight after popping blocks
+
+\[bu] \fBstatus\fR - string; General RPC error code. "OK" means everything looks good.
+
+\[bu] \fBuntrusted\fR - boolean; States if the result is obtained using the bootstrap mode, and is therefore not trusted (\fBtrue\fR), or when the daemon is fully synced and thus handles the RPC locally (\fBfalse\fR)
+
+
+.TP
+.B Example:
+
+.nf
+.RS 4
+$ curl http://127.0.0.1:18081/pop_blocks -d '{"nblocks":6}' -H 'Content-Type: application/json''
+
+{
+  "height": 76482,
+  "status": "OK",
+  "untrusted": false
+}
+
+.RE
+.fi
+
+
+
+
+
+.SH EXAMPLES
+
+.SS Change Log file:
+Full path to the log file. Example (mind file permissions):
+
+.B ./monerod --log-file=/var/log/monero/mainnet/monerod.log
+
+.SS Change PID:
+Full path to the PID file. Works only with --detach.
+
+.B ./monerod --detach --pidfile=/run/monero/monerod.pid
+
+.SS Pick Blockchain:
+
+You can run \fBmonerod\fR in two modes,
+.sp 2 
+stagenet for learning and experimentation.
+
+\fB\./monerod \--stagenet \--detach\fR        # run as a daemon in background
+.br
+\fBtail \-f ~/.bitmonero/stagenet/bitmonero.log\fR  # watch the logs
+.br
+\fB\./monerod \--stagenet exit\fR           # ask daemon to exit gracefully
+
+mainnet is when you want to deal with the real XMR.
+
+\fB\./monerod \--detach\fR        # run as a daemon in background
+.br
+\fBtail \-f ~/.bitmonero/bitmonero.log\fR  # watch the logs
+.br
+\fB\./monerod exit\fR           # ask daemon to exit gracefully
+
+
+.SS p2p network protocol over Tor via torsocks:
```

Comment:
Most conventions would consider `P2P` an acronym and capitalize it throughout the document; I don't mind keeping it lower case, but, as the first word in a section heading, it should have the first letter capitalized (`P2p`)

Reply:
Note: The prior comment applies to the following section, as well.